src/battle-log.ts

```diff
@@ -141,9 +141,9 @@ class BattleLog {
 
 		case 'unlink': {
 			const user = toID(args[2]) || toID(args[1]);
-			this.unlinkChatFrom(user);
+			this.unlinkChatFrom(user, parseInt(args[args.length], 10));
```

Comment:
```suggestion
			this.unlinkChatFrom(user);
```

Reply:
Hiding a line count only matters for `|unlink|hide|`, so you don't have to deal with this if it's _not_ a hide instruction.

You didn't change the `unlinkChatFrom` function to take a line count in the first place.